src/Codeception/Lib/Driver/Db.php

```diff
@@ -221,7 +221,14 @@ public function executeQuery($query, array $params)
             throw new \Exception("Query '$query' can't be prepared.");
         }
 
-        $sth->execute($params);
+        $i = 0;
+        foreach ($params as $value) {
+            $i++;
+            $type = preg_match('/^[-+]?\d+$/', $value) ? \PDO::PARAM_INT : \PDO::PARAM_STR;
```

Comment:
Leaves them as strings which should never be an issue. If you [refer to the issue I opened](https://github.com/Codeception/Codeception/issues/3118), this is really only enabling read/write access to `BIT(1)` fields which give a "data too long" error when passing a string parameter.